### ISSUE TITLE
Mistakes on Vessel page [DOCUMENTATION]

### ISSUE THREAD

**Issue by nuggreat** [KSP-KOS/KOS#2982]
There is a duplicate entry for the suffix `:DELTAV` one for the initial implementation that never when to release where it simply returned a scalar with the total Dv of the vessel and another that returns the deltaV structure.  The suffix table also contains entries for both 

Also the suffixes `:BURNTIME`, `:DELTAVASL`, and `:DELTAVVACUUM` are listed as existing on vessels they but they where removed before release as the Dv data was moved into it's own structure.

**Comment by mgalyean** [comment]
SHIP:DELTAV:DURATION look like it does BURNTIME now

**Comment by nuggreat** [comment]
> 
> 
> SHIP:DELTAV:DURATION look like it does BURNTIME now

Yes I know this is more about the incorrect suffixes listed as being available for query on a vessel.  As I don't need to know this but the listing of incorrect suffixes has tripped up at least one user already which is what prompted me to make the issue.

**Comment by mgalyean** [comment]
Yeah, I threw that in there for anyone who ended up here prior to the next release so they might get a script working

**Comment by nuggreat** [comment]
Fixed by PR #3137
